ipfssync/ipfssync.go

```diff
@@ -69,26 +68,20 @@ type IPFSsync struct {
 func NewIPFSsync(dataDir, groupKey, privKeyHex, transport string, storage data.Storage) *IPFSsync {
 	is := &IPFSsync{
 		DataDir:         dataDir,
-		Topic:           groupKey,
+		GroupKey:        groupKey,
 		PrivKey:         privKeyHex,
 		Port:            4171,
 		HelloTime:       40,
 		UpdateTime:      20,
```

Comment:
while you're here, might as well make these proper `time.Duration` types. and I think the names should be `UpdateInterval` and such, to clarify they are intervals, and not specific timestamps.

Reply:
https://github.com/vocdoni/vocdoni-node/pull/363/commits/e31f7af58c36c16df22a671a124c577355393c4c